Updated the link

Prabhat-IIT opened the pull request:
The preivious markdown link was problematic. So, instead of it a http link is added. @jgbarah Its the required fix for the issue I recently reported in the grimoirelab tutorial.
I was not able to fix this using only markdown because github doesn't support opening links in new tab
[Stackoverflow](https://stackoverflow.com/questions/41915571/open-link-in-new-tab-with-github-markdown-using-target-blank)
[Github open issue](https://github.com/mojombo/github-flavored-markdown/issues/28)

Prabhat-IIT commented:
Actually, relatives links won't work here.We need to open the link in a new tab for successful download. Also, I couldn't find a way to do this in markdown itself as Github doesn't support it. Although let me see if something can be done regarding relative links. Do you have something in mind?

Prabhat-IIT commented:
I've tested with a relative hyperlink the same problem will come in that. What do you suggest?

jgbarah commented:
> Actually, relatives links won't work here.We need to open the link in a new tab for successful download. Also, I couldn't find a way to do this in markdown itself as Github doesn't support it. Although let me see if something can be done regarding relative links. Do you have something in mind? Moreover, these links would still work if one gets his/her own gitbook

I think you don't need to open the file in a new tab. If clicking on the link just opens the file, that's fine. Depending on your browser and your configuration, very likely instead of opening it the browser would offer to download it. And you can always right-click, and then select download...

My problem is that I couldn't realize a way of writing the relative link so that it works....

jgbarah commented:
OK, now I understand what you mean. Thanks for your patience explaining! I had never "just clicked" on the link, I had always used "save as". Now, I see that if you just click, nothing happens, which I agree, is annoying.

OTOH, I would prefer to avoid absolute links as much as possible. What do you think about adding a notice next to the link, something like "(use "save as", or open in another tab)"?

Prabhat-IIT commented:
I have added the Save as warning and removed the absolute links. I've not added the suggestion for opening in new tab because if the user clicked on the link once simply and then tried to open in new tab then the new tab download won't work due to bad end point.

Prabhat-IIT commented:
can it be merged now @jgbarah  ?

Prabhat-IIT commented:
Thanks @jgbarah  :)
